tests/sentry/grouping/test_builtin_fingerprinting.py

```diff
@@ -25,11 +27,11 @@
 
 
 @pytest.fixture
-def default_bases():
+def default_bases() -> list[str]:
     return ["javascript@2024-02-02"]
 
 
-def test_default_bases(default_bases):
+def test_default_bases(default_bases: Sequence[str]) -> None:
```

Comment:
Any percentage in making the `default_bases` type here match the type it's actually typed with above? WDYT?

```suggestion
def test_default_bases(default_bases: list[str]) -> None:
```